src/BlazorUI/Bit.BlazorUI/Extensions/StringExtensions.cs

```diff
@@ -1,15 +1,18 @@
-﻿namespace Bit.BlazorUI;
+﻿using System.Runtime.CompilerServices;
 
-public static class StringExtensions
+[assembly: InternalsVisibleTo("Bit.BlazorUI.Playground.Web")]
```

Comment:
since this is an assembly level attribute, you need to apply it once.
please remove this duplicate line.